packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,224 @@
+import { TextField } from "@kaizen/draft-form"
+import React, { RefObject, useEffect, useRef, useState } from "react"
+import dateStart from "@kaizen/component-library/icons/date-start.icon.svg"
+import "react-day-picker/lib/style.css"
+import DayPicker from "react-day-picker/DayPicker"
+import { usePopper } from "react-popper"
+import cx from "classnames"
+import {
+  DayModifiers,
+  RangeModifier,
+  BeforeAfterModifier,
+} from "react-day-picker/types/Modifiers"
+import { daysToNumbers } from "../utils/daysToNumbers"
+import { useClickOutside } from "../hooks/useClickOutside"
+import datePickerStyles from "./DatePicker.scss"
+import { CalendarNav, CalendarNavProps } from "./CalendarNav"
+import { defaultDatePickerClasses } from "./DatePickerClasses"
+
+interface DatePickerProps {
+  id: string
+  classNameAndIHaveSpokenToDST?: string
+  labelText?: string
+  isDisabled?: boolean
+  inputRef?: RefObject<HTMLInputElement> | undefined
+  description?: string
+
+  /** Accepts a DayOfWeek value to start the week on that day. By default,
+   * it's set to Monday.
+   */
+  firstDayOfWeek?: DayOfWeek
+
+  // Accepts a date to display that month on first render.
+  initialMonth?: Date
+
+  // The date passed in from the consumer that renders in the input and calendar.
+  selectedDate?: Date
+
+  // Event passed from consumer to handle the date on change.
+  onDayChange: (day: Date) => void
```

Comment:
Similar to above, we might want this to just be `onChange` for consistency with our other components.